### DISCUSSION TITLE
What country is home to the Matterhorn?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#402] [Galaxy Trivia] [answered]
**Hint:** it's tricky

**Comment by peterbaccaro** [comment]
The Matterhorn is a famous Swiss landmark however it is situated on the border between **Switzerland** and **Italy** and yet its summit is part of **Africa**.

The Alps were formed about 30 million years ago, when the African tectonic plate collided from the south into the European tectonic plate. Geologists studying the Matterhorn have found that the lower 11,000 feet of its slopes are sedimentary rock from the European plate and ocean crust from forgotten seas.  But the top 3,000 feet—the mountain's peak—is actually harder and older metamorphic rock from the African plate, pushed on top of the European sediment and gradually eroded away by glaciers. So most of the Matterhorn is a European mountain—but the top is technically African.

**Comment by thuongtruong1009** [comment, marked as answer]
The Matterhorn is a mountain peak located in the Swiss Alps. Therefore, the country that is home to the Matterhorn is Switzerland. The mountain is situated on the border between Switzerland and Italy, but its summit is within Swiss territory. The Matterhorn is one of the most famous and iconic mountains in the world and is a popular destination for hikers, climbers, and skiers. Its distinctive pyramid shape and challenging climbing routes have made it a symbol of Swiss tourism and a magnet for adventurers from around the world.
